pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -980,6 +1018,8 @@ def _checkURL(self, inputcheck, combo):
             protocol = urlsplit(repo.baseurl)[0] # extract protocol part (http, https, nfs...)
             if repo.mirrorlist:
                 url_string = repo.mirrorlist.strip()[len(protocol) + 3:] # +3 for "://" part
+            elif repo.metalink:
+                url_string = repo.metalink.strip()[len(protocol) + 3:] # +3 for "://" part
```

Comment:
Please extract his repetition to a separate method.

Reply:
Will do.